<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="Menu" native="true">
     <property name="minimumSize">
      <size>
       <width>66</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>66</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>8</number>
      </property>
      <property name="topMargin">
       <number>8</number>
      </property>
      <property name="rightMargin">
       <number>8</number>
      </property>
      <property name="bottomMargin">
       <number>8</number>
      </property>
      <item>
       <widget class="Button" name="menubar_btnMain">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Button" name="menubar_btnContacts">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Button" name="menubar_btnMore">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="menubar_spacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="Button" name="menubar_btnSettings">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="mainContent" native="true">
     <widget class="MainContainer" name="mainContainer" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>481</width>
        <height>411</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <widget class="PageChats" name="page_1" native="true">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>40</y>
         <width>341</width>
         <height>80</height>
        </rect>
       </property>
      </widget>
      <widget class="PageContacts" name="page_2" native="true">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>120</y>
         <width>341</width>
         <height>80</height>
        </rect>
       </property>
      </widget>
      <widget class="QWidget" name="page_3" native="true">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>200</y>
         <width>341</width>
         <height>80</height>
        </rect>
       </property>
      </widget>
      <widget class="PageSettings" name="page_4" native="true">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>280</y>
         <width>351</width>
         <height>80</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Button</class>
   <extends>QPushButton</extends>
   <header>uicomponent/Button.h</header>
  </customwidget>
  <customwidget>
   <class>MainContainer</class>
   <extends>QWidget</extends>
   <header>uicomponent/MainContainer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PageChats</class>
   <extends>QWidget</extends>
   <header>ui/PageChats.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PageContacts</class>
   <extends>QWidget</extends>
   <header>ui/PageContacts.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PageSettings</class>
   <extends>QWidget</extends>
   <header>ui/PageSettings.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
